src/libs/OptionsListUtils.ts

```diff
@@ -96,81 +154,80 @@ Onyx.connect({
- * @return {String}
  */
-function addSMSDomainIfPhoneNumber(login) {
+function addSMSDomainIfPhoneNumber(login: string): string {
     const parsedPhoneNumber = parsePhoneNumber(login);
     if (parsedPhoneNumber.possible && !Str.isValidEmail(login)) {
-        return parsedPhoneNumber.number.e164 + CONST.SMS.DOMAIN;
+        return parsedPhoneNumber.number?.e164 + CONST.SMS.DOMAIN;
     }
     return login;
 }
 
 /**
- * Returns avatar data for a list of user accountIDs
- *
- * @param {Array<Number>} accountIDs
- * @param {Object} personalDetails
- * @param {Object} defaultValues {login: accountID} In workspace invite page, when new user is added we pass available data to opt in
- * @returns {Object}
+ * @param defaultValues {login: accountID} In workspace invite page, when new user is added we pass available data to opt in
+ * @returns Returns avatar data for a list of user accountIDs
  */
-function getAvatarsForAccountIDs(accountIDs, personalDetails, defaultValues = {}) {
-    const reversedDefaultValues = {};
-    _.map(Object.entries(defaultValues), (item) => {
+function getAvatarsForAccountIDs(accountIDs: number[], personalDetails: OnyxEntry<PersonalDetailsList>, defaultValues: Record<string, number> = {}): OnyxCommon.Icon[] {
+    const reversedDefaultValues: Record<number, string> = {};
+
+    Object.entries(defaultValues).forEach((item) => {
         reversedDefaultValues[item[1]] = item[0];
     });
-
-    return _.map(accountIDs, (accountID) => {
-        const login = lodashGet(reversedDefaultValues, accountID, '');
-        const userPersonalDetail = lodashGet(personalDetails, accountID, {login, accountID, avatar: ''});
+    return accountIDs.map((accountID) => {
+        const login = reversedDefaultValues[accountID] ?? '';
+        const userPersonalDetail = personalDetails?.[accountID] ?? {login, accountID, avatar: ''};
 
         return {
             id: accountID,
             source: UserUtils.getAvatar(userPersonalDetail.avatar, userPersonalDetail.accountID),
             type: CONST.ICON_TYPE_AVATAR,
-            name: userPersonalDetail.login,
+            name: userPersonalDetail.login ?? '',
         };
     });
 }
 
 /**
  * Returns the personal details for an array of accountIDs
- *
- * @param {Array} accountIDs
- * @param {Object | null} personalDetails
- * @returns {Object} – keys of the object are emails, values are PersonalDetails objects.
+ * @returns  keys of the object are emails, values are PersonalDetails objects.
  */
-function getPersonalDetailsForAccountIDs(accountIDs, personalDetails) {
-    const personalDetailsForAccountIDs = {};
+function getPersonalDetailsForAccountIDs(accountIDs: number[], personalDetails: OnyxEntry<PersonalDetailsList>): PersonalDetailsList {
+    const personalDetailsForAccountIDs: PersonalDetailsList = {};
     if (!personalDetails) {
         return personalDetailsForAccountIDs;
     }
-    _.each(accountIDs, (accountID) => {
+    accountIDs?.forEach((accountID) => {
```

Comment:
Optional chaining (?) is not necessary here since `accountIDs` is defined as `number[]`